client-app-example/src/client.js

```diff
@@ -0,0 +1,61 @@
+// Load polyfills required for IE11
+import '@babel/polyfill';
+import 'custom-event-polyfill/polyfill.js';
+import 'url-polyfill';
+
+import { Client } from 'iframe-coordinator/dist/client';
+
+document.getElementById('path').innerHTML = window.location.hash;
+
+window.onhashchange = function() {
+  document.getElementById('path').innerHTML = window.location.hash;
+};
+
+// Start intercepting link click events for routing
+let iframeClient = new Client({
+  // This lets the client work correctly with the cli host
+  hostOrigin: 'http://localhost:3000'
+});
+iframeClient.addListener('environmentalData', envData => {
+  document.getElementById('env-data').innerHTML = JSON.stringify(
```

Comment:
It might be cool, to show us setting the Intl API to the locale or something to show some real examples of what needs to be done.

Reply:
That's a pretty good idea. I'll add something like that.